### PR TITLE
[ANT-24] Add orgs Ghent/Antwerp and Summit

### PR DESCRIPTION
**Pull request by FloorD**
Update Organizers lists for Ghent/Antwerp and the Organizer Summit. Now with photos. Asking in channels for feedback.

### CODE REVIEW

**Review by somatorio — APPROVED** [review]

File: static/events/2024-ghent/organizers/mona (1).jpeg
Comment (yvovandoorn) [inline static/events/2024-ghent/organizers/mona (1).jpeg:1]: why are we adding a file with `(1)` in the file name?

File: data/events/2024-organiser-summit.yml [hunk 1]
```diff
@@ -53,15 +53,67 @@ team_members: # Name is the only required field for team members.
...
+  - name: "JJ Asghar"
+    twitter: "jjasghar"
+    github: "jjasghar"
+    image: "jj.jpg"
+  - name: "Mona Shivdasani"
+    pronouns: "she/her"
+    linkedin: "https://www.linkedin.com/in/monashivdasani/"
+    twitter: "TrilandsBV"
+    github: "MonaShivdasani"
+    image: "mona (1).jpeg"
```
Comment (yvovandoorn) [inline data/events/2024-organiser-summit.yml:105]: why are we adding file with `(1)` in the filename

File: data/events/2024-ghent.yml [hunk 2]
```diff
@@ -50,50 +50,67 @@ nav_elements: # List of pages you want to show up in the navigation of your page
 
 # These are the same people you have on the mailing list and Slack channel.
 team_members: # Name is the only required field for team members.
-  - name: "Kris Buytaert"
+- name: "Kris Buytaert"
```
Comment (zipkid) [inline data/events/2024-ghent.yml:53]: Missing an indent here.

File: static/events/2024-ghent/organizers/mona (1).jpeg
Comment (FloorD) [reply static/events/2024-ghent/organizers/mona (1).jpeg:1]: copied from Amsterdam repo, agree it's not pretty but it works

**Review by ringods — APPROVED** [review]

File: static/events/2024-ghent/organizers/mona (1).jpeg
Comment (mattstratton) [reply static/events/2024-ghent/organizers/mona (1).jpeg:1]: Can I ask very nicely for you to change them to not have spaces in them? I have some planned maintenance tasks for improvement that I know will break with things like this, and while there are existing files that I'll have to deal with, I am very politely asking you to not make there be more of them :-)

**Pull request merged**